Dockerfile missing temporal-network, image temporalio/admin-tools does not supports version 1.23

Hey, I am new to Open Source and would like to contribute, I want to get started with the issues list, but got stuck in the beginning itself, I tried the documentation mentioned steps and got some errors as mentioned below, I guess this is a valid issue, your help and support is highly appreciated, Thanks

I had raised the PR's with the below fix, do let me know if anything is wrong.
https://github.com/nucleuscloud/neosync/pull/2355


## Bug description

## How to reproduce

1. just do `make `compose/up` in the cloned repo

after the above step, I received two errors one after the another, first error was

docker compose -f compose.yml up -d
service "api" refers to undefined network temporal-network: invalid compose project
make: *** [compose/up] Error 15

this was fixed by adding the temporal-network in the network section of compose.yml

```
temporal-network:
    name: temporal-network
```

after this I did the `make compose/up` again and received another error as,

 ✘ temporal-seed Error                                                                                                       3.9s 
Error response from daemon: manifest for temporalio/admin-tools:1.23 not found: manifest unknown: manifest unknown
make: *** [compose/up] Error 18

that too can be fixed by changing the version of  `temporalio/admin-tools` to some other tag, I tested this with `latest` and `1.23.1.1-tctl-1.18.1-cli-0.12.0` this both worked, and made the app up, I have committed the version as `1.23.1.1-tctl-1.18.1-cli-0.12.0` 


## Environment
Docker version 24.0.5, build ced0996600
OS : Sonoma 14.5 (Mac M1)


#### *Thank you* for your bug report – we love squashing them!

--- a/compose.yml
+++ b/compose.yml
@@ -96,7 +96,7 @@ services:
 
   temporal-seed:
     container_name: neosync-temporal-seed
-    image: temporalio/admin-tools:1.23
+    image: temporalio/admin-tools:1.23.0
     environment:
       - TEMPORAL_ADDRESS=temporal:7233
     entrypoint: /bin/sh -c "/create_schedules.sh"
@@ -141,3 +141,5 @@ volumes:
 networks:
   neosync-network:
     name: neosync-network
+  temporal-network:
+    name: temporal-network

--- a/compose.yml
+++ b/compose.yml
@@ -141,5 +141,3 @@ volumes:
 networks:
   neosync-network:
     name: neosync-network
-  temporal-network:
-    name: temporal-network

